maths/entropy.py

```diff
@@ -0,0 +1,119 @@
+"""
+    Implementation of entropy of information
+    https://en.wikipedia.org/wiki/Entropy_(information_theory)
+"""
+
+import sys
+import math
+r = range
+
+
+def create_alpha_array():
+    """
+    This method creates array of our alphabet to print probabilities later on.
+    :return:
+    """
+    my_alphas = [" "]
+    for x in r(ord("a"), ord("z") + 1):
+        my_alphas += chr(x)
+    return my_alphas
+
+
+def calculate_prob(my_dic_fir, my_dic_sec, path_dest):
+    """
+    This method takes path and two dict as argument
+    and than calculates entropy of them.
+    :param my_dic_fir:
+    :param my_dic_sec:
+    :param path_dest:
+    :return:
+    """
+    f = open(path_dest, mode="w+", encoding="utf-8")
+
+    my_alphas = create_alpha_array()  # create our alpha
+    # what is our total sum of probabilities.
+    values = my_dic_fir.values()
+    all_sum = sum(values)
+
+    # one length string
+    my_fir_sum = 0
+    # for each alpha we go in our dict and if it is in it we calculate entropy
+    for _ in my_alphas:
```

Comment:
Only use _ as a variable name if you are never going to refer to it again.  In this case, please replace _ with `char`.

Or better yet, use `for key, value in first_dict.items():`.  The use of `my_` in variable names is not helpful.